include/uv.h

```diff
@@ -475,7 +475,7 @@ UV_EXTERN uv_buf_t uv_buf_init(char* base, unsigned int len);
 
 #define UV_STREAM_FIELDS                                                      \
   /* number of bytes queued for writing */                                    \
-  size_t write_queue_size;                                                    \
+  uint64_t write_queue_size;                                                  \
```

Comment:
This breaks ABI on 32-bits architectures, as does changing the return type of `uv_stream_get_write_queue_size()`.